.circleci/config.yml

```diff
@@ -235,9 +241,11 @@ references:
         docker_image_tag=${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:app-${CIRCLE_SHA1}
         kubectl apply -f kubernetes_deploy/production/secrets.yaml
         kubectl set image -f kubernetes_deploy/production/deployment.yaml cccd-app=${docker_image_tag} --local -o yaml | kubectl apply -f -
+        kubectl set image -f kubernetes_deploy/cron_jobs/archive_stale.yaml cronjob-worker=${docker_image_tag} --local -o yaml | kubectl apply -f -
         kubectl apply \
         -f kubernetes_deploy/production/service.yaml \
-        -f kubernetes_deploy/production/ingress.yaml
+        -f kubernetes_deploy/production/ingress.yaml \
+        -f kubernetes_deploy/cron_jobs/clean_ecr.yaml
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3074/files#r345769551)